ui/server/logback-dev.xml

```diff
@@ -0,0 +1,16 @@
+<configuration scan="true" scanPeriod="5 seconds">
+    <jmxConfigurator/>
+
+    <appender name="stdout" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <Pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</Pattern>
+        </encoder>
+    </appender>
+
+    <root level="debug">
```

Comment:
Root logger on "debug" is not a good idea. We use libs that send many many logs with debug level. Better option will be info or even warn. Debug can be for `pl.touk.nussknacker`